### ISSUE TITLE
@license spdx ignore for private modules

### ISSUE THREAD

**Issue by dennismphil** [gajus/eslint-plugin-jsdoc#884]
See some discussions here https://github.com/npm/npm/issues/8773

## Expected behavior

```ts
// code.js
/**
 * @license
 * Copyright 2022 Acme, Corp. All Rights Reserved.
 */

// package.json
{
   ...
   private: true,
   ...
}
```

shows a warning 
```sh
Invalid JSDoc @license: "Copyright 2022 Acme, Corp. All Rights Reserved."; expected SPDX expression: https://spdx.org/licenses/.eslint[jsdoc/check-values](https://github.com/gajus/eslint-plugin-jsdoc#eslint-plugin-jsdoc-rules-check-values)
```

## Actual behavior

```ts
/**
 * @license
 * Copyright 2022 Acme, Corp. All Rights Reserved.
 */
```
this should not show a warning

## Environment

- Node version: v17.9.0  
- ESLint version: v8.12.0
- `eslint-plugin-jsdoc` version: 39.2.8

**Comment by brettz9** [comment]
You can use the `licensePattern` option to indicate where any valid expression (including an empty one) can be found.

See the tests added in f4a92498190ca530dbc17d58c2514ac102ab719a (or look at the updated README).

The just released v39.2.9 improves the behavior of the option (to ensure it doesn't fail when there isn't a match), so you'll probably want to update to it.

That should address your question, I think, but feel free to comment further as needed.
